### DISCUSSION TITLE
AssertionError, appeared from no where.

### DISCUSSION THREAD

**Discussion by mchingotto** [netbox-community/netbox#12816] [Q&A]
Hello guys,

   This message started to appear few hours ago, both in dev and production when We want to create a device-type.
   It is weird because both installs are isolated.

Any clue ?
Thank you, I appreciate it.

--------------------------------------------------------
<class 'AssertionError'>

Invalid attribute 'name' for DeviceType

Python version: 3.9.9
NetBox version: 3.2.0-beta2

--------------------------------------------------------

**Comment by DanSheps** [comment]
It looks like someone may have placed a validator in your settings to check for the presence of a attribute with the name "name", based on the error itself.  I would check your validation rules you have loaded into NetBox.

(DeviceType doesn't have "name", it has "model" and "part_number")

**Comment by mchingotto** [comment]
Hello Dan,

   That's was exactly what happened ! Thank you for your fast answer. Sorry for asking also, I was in a hospital waiting for someone and my head in other place. I got to the same idea minutes after asking.
   Maybe could be good to validate the keys from the validators before to apply to the models, so the error does not appear as AssertionError, maybe could be silently avoided instead.
